tests/laser/state/calldata_test.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+from mythril.laser.ethereum.state import Calldata
+from z3 import Solver, simplify
+from z3.z3types import Z3Exception
+
+
+
+uninitialized_test_data = [
+    (None), # Symbolic calldata
+    ([]), # Empty concrete calldata
+    ([1,4,5,3,4,72,230,53]) # Concrete calldata
+]
+@pytest.mark.parametrize("starting_calldata", uninitialized_test_data)
+def test_calldata_uninitialized_index(starting_calldata):
+    # Arrange
+    calldata = Calldata(0, starting_calldata)
+    solver = Solver()
+
+    # Act
+    value = calldata[100]
+
+    solver.add(calldata.constraints)
+    solver.check()
+    model = solver.model()
+
+    value = model.eval(value)
+
+    # Assert
+    assert value == 0
```

Comment:
why is this 0 in the symbolic calldata situation?

Reply:
Because I forgot to delete a line. :(